doc/introduction/measurements.rst

```diff
@@ -8,23 +8,21 @@ Measurements
 
 .. currentmodule:: pennylane.measure
 
-PennyLane can extract different types of measurement results: The expectation of an observable
-over multiple measurements, its variance, or a sample of a single measurement.
+PennyLane can extract different types of measurement results: the expectation of an observable
+over multiple measurements, its variance, or samples of a single measurement.
 
 For example, the quantum function shown in the previous section
```

Comment:
I think it's better if we minimize these sequential references and make every page standalone (even if content is repeated). Users could land on this page as the first place they click. (also makes it more maintainable if the other page later changes)

Reply:
Agree, will change.